gpustat.py

```diff
@@ -434,6 +443,8 @@ def main():
                         help='Print all the information in JSON format')
     parser.add_argument('-v', '--version', action='version',
                         version=('gpustat %s' % __version__))
+    parser.add_argument('--ret-json', action='store_true', default=False,
+                        help='return json for calling as library')
```

Comment:
I don't think this argument is necessary. It looks somewhat misleading; for the sake of getting the object, we can just call the function or parse the result.

Reply:
I made a new function and call that function to give me the json.